lib/explorer/backend/lazy_series.ex

```diff
@@ -549,6 +558,61 @@ defmodule Explorer.Backend.LazySeries do
     """
   end
 
+  @impl true
+  def contains(series, %Regex{source: pattern}) do
+    data = new(:contains, [lazy_series!(series), pattern])
+
+    Backend.Series.new(data, :boolean)
+  end
+
+  def contains(series, pattern) when is_binary(pattern) do
+    data = new(:contains_literal, [lazy_series!(series), pattern])
+
+    Backend.Series.new(data, :boolean)
+  end
+
+  @impl true
+  def upcase(series) do
+    data = new(:to_uppercase, [lazy_series!(series)])
```

Comment:
LazySeries is `polars` agnostic. So we should stick with Explorer names instead of Polars ones. Same applies below!

Reply:
Good point. I'll fix them up.